Topics/Teamwork.md

```diff
@@ -1,4 +1,26 @@
-## Resources for Teamwork
+# Resources for Teamwork
+
+## Forming a Team
+When first forming a team, it is important for all members to get to know one another as this will allow each member to feel more comfortable communicating while working together. It is highly recommended that teams meet prior to starting the project to simply get aquainted with one another. Although this meeting should be more personal (i.e. unrelated to the project), it can also be used to establish the group's goals and expectations to ensure everyone has a similar mindset going into the project. 
+
+It is also extremely useful for team members to identify the specific skill sets that they can bring to the team, including any relevant soft skills and technical experience. These strengths should be shared with the team to help with the decisions making when delegating roles and responsibilities. 
+
+## Project Management
+Working as a team effectively requires a lot of planning and organization. Projects as a whole are often very complicated and overwhelming, so it is very useful to break down a project into smaller stages/tasks. There are many ways in which a project can be managed; two commons project management practices that software engineering teams user are the Waterfall and Agile methods. Details on these can be found in [Resources for Software Engineers](https://learning-software-engineering.github.io/Topics/Software_Engineering.html#resources-for-software-engineering).
+
+## Team Communication
+During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex. discord or slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new reponsibilities, and set project deadlines. 
```

Comment:
```suggestion
During the early stages of forming a team, the group should also establish general expectations for how the team plans to communicate with one another. This may include setting up a team communication channel on a specific platform (ex., Discord or Slack) and establishing regular check-up meetings (including when these meetings are and where they will be held - for example, in person or online). These general meetings can be used to outline general expectations and project requirements, update one another on individual progress, delegate new responsibilities, and set project deadlines. 
```